core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -670,7 +670,7 @@ private[spark] class MapOutputTrackerMaster(
 /**
  * Executor-side client for fetching map output info from the driver's MapOutputTrackerMaster.
  * Note that this is not used in local-mode; instead, local-mode Executors access the
- * MapOutputTrackerMaster directly (which is possible because the master and worker share a comon
+ * MapOutputTrackerMaster directly (which is possible because the master and worker share a common
```

Comment:
Could you check more files please? Usually, it would be very helpful if you finish module by module. Maybe, `core` module in this case?

Reply:
Ok. Thx for your replies, I will inspect the typo mistakes in `core` module later.